docker-entrypoint.d/00-update-uid

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# current www-data UID
+CURRENT_UID=`id -u www-data`
+WANTED_UID=${RUN_AS_UID:-33}
+
+# check if UID need to be updated
+if [ $CURRENT_UID -ne $WANTED_UID ]; then
```

Comment:
I tried using this same code in a different docker image to change the `www-data` UID and needed to tweak this line to `if [ "$CURRENT_UID" -ne "$WANTED_UID" ]; then`, and the above to `WANTED_UID=${RUN_AS_UID}`